app/routers/model_router.py

```diff
@@ -3,21 +3,32 @@
 from fastapi import APIRouter
 
 from app.data import MongoDB
-from app.model import MatcherSortSearch
+from app.model import MenteeMatcherSearch
+from app.model import MentorMatcherSearch
 
 Router = APIRouter(
     tags=["Model Operations"],
 )
 Router.db = MongoDB()
-Router.matcher = MatcherSortSearch()
+Router.mentee_matcher = MenteeMatcherSearch()
+Router.mentor_matcher = MentorMatcherSearch()
 
+@Router.post("/mentee_match/{profile_id}")
```

Comment:
In URLs it is best practice to use the `-` dash character as a word separator rather than the Pythonic `_` underscore. This is not a huge problem, but we want our API to be consistent, double check the other endpoints.